src/shortestpaths/dijkstra.jl

```diff
@@ -36,7 +36,7 @@ function dijkstra_shortest_paths(
     preds = fill(Vector{U}(), nvg)
     visited = zeros(Bool, nvg)
     pathcounts = zeros(Int, nvg)
-    H = PriorityQueue{U,T,Base.Order.ForwardOrdering}() # this should be Vector{T}() in 0.4, I think.
+    H = PriorityQueue{U,T}(Base.Order.ForwardOrdering())
```

Comment:
Since forward ordering is the default, this can be 
```julia
PriorityQueue{U,T}()
```

If you really want to have the Ordering object there, then it should probably be

```julia
H = PriorityQueue{U,T}(Base.Order.Forward)
```

To match the `Reverse` syntax in madadjvisit.jl

Reply:
Also, this should be the only line that actually broke in your code (sorry about that!), and should be fixed by https://github.com/JuliaCollections/DataStructures.jl/pull/326 (with a deprecation warning).

Reply:
Yup. It'll be in the next release. Thanks for your help.